include/apple2.h

```diff
@@ -83,7 +83,63 @@
 #define CH_CURS_LEFT    0x08
 #define CH_CURS_RIGHT   0x15
 
-#if !defined(__APPLE2ENH__)
+#if defined(__APPLE2ENH__)
+
+/* MouseText-based functions for boxes and lines drawing */
+void mt_chline(unsigned char length);
+void mt_cvline(unsigned char length);
+void mt_chlinexy(unsigned char x, unsigned char y, unsigned char length);
+void mt_cvlinexy(unsigned char x, unsigned char y, unsigned char length);
+
+#define CH_HLINE        0x5F
+#define CH_VLINE        0xDF
+#define CH_ULCORNER     0x5F
+#define CH_URCORNER     0x20
+#define CH_LLCORNER     0xD4
+#define CH_LRCORNER     0xDF
+#define CH_TTEE         0x5F
+#define CH_BTEE         0xD4
+#define CH_LTEE         0xD4
+#define CH_RTEE         0xDF
+#define CH_CROSS        0xD4
+
+#define _chline(length)       mt_chline(length)
+#define _chlinexy(x,y,length) mt_chlinexy(x,y,length)
+#define _cvline(length)       mt_cvline(length)
+#define _cvlinexy(x,y,length) mt_cvlinexy(x,y,length)
+
+#else
+
+/* Functions that don't depend on MouseText to draw boxes and lines */
+void dyn_chline(unsigned char h, unsigned char length);
+void dyn_cvline(unsigned char v, unsigned char length);
+void dyn_chlinexy(unsigned char h, unsigned char x, unsigned char y, unsigned char length);
+void dyn_cvlinexy(unsigned char v, unsigned char x, unsigned char y, unsigned char length);
+
+#if defined(DYN_BOX_DRAW)
+/* When the user defines DYN_BOX_DRAW, we'll adapt to the machine
+** we run on.
+ */
+extern char CH_HLINE;
+extern char CH_VLINE;
+extern char CH_HLINE_SCRCODE;
+extern char CH_VLINE_SCRCODE;
+extern char CH_ULCORNER;
+extern char CH_URCORNER;
+extern char CH_LLCORNER;
+extern char CH_LRCORNER;
+extern char CH_TTEE;
+extern char CH_BTEE;
+extern char CH_LTEE;
+extern char CH_RTEE;
+extern char CH_CROSS;
```

Comment:
I don't know why we're back to 13 individual chars. I seem to remember we had agreed that we need only 5. In a comment to the old PR I had already shown how the 13 macros map to the 5 values.

Reply:
I've done it this way but they point to 5 different bytes (see libsrc/apple2/boxchars.s)